src/pke/lib/schemerns/rns-multiparty.cpp

```diff
@@ -146,6 +152,12 @@ Ciphertext<DCRTPoly> MultipartyRNS::MultipartyDecryptMain(ConstCiphertext<DCRTPo
 
         noise = e;
     }
+    else if (cryptoParams->GetDecryptionNoiseMode() == NOISE_FLOODING_DECRYPT &&
+             cryptoParams->GetExecutionMode() == EXEC_EVALUATION) {
+        auto dgg = cryptoParams->GetFloodingDiscreteGaussianGenerator();
+        DCRTPoly e(dgg, cv[0].GetParams(), Format::EVALUATION);
```

Comment:
the move assignment could be used here and for "else" in this if-else block:
noise = std::move(DCRTPoly (dgg, cv[0].GetParams(), Format::EVALUATION));

Reply:
Done

Reply:
The above didn't work with clang, so I changed it to:
DCRTPoly e(dgg, cv[0].GetParams(), Format::EVALUATION);
noise = std::move(e);